### ISSUE TITLE
Export layers to files

### ISSUE THREAD

**Issue by dsizzle** [HaikuArchives/ArtPaint#460]
It might be handy to have an option to export individual layers to files, or all layers.  It'd be at least a step towards animated GIFs or other animations to be able to spit out all layers to files with a base name and numbers, like "anim000.png", or perhaps you might want a single layer exported.  

Related to #37

**Comment by humdingerb** [comment]
Yes. Maybe a checkbox in the "Save image..." dialog: "One file per visible layer"
